Resources/public/js/views/ez-usermenuview.js

```diff
@@ -0,0 +1,170 @@
+            this.containerTemplate = '<ul class="ez-view-usermenuview"/>';
+
+            this.on('displayedChange', this._toggleUserMenu);
+            this.after('displayedChange', this._setClickOutsideUserMenuHandler);
+        },
+
+        /**
+         * Renders the user menu view
+         *
+         * @method render
+         * @return {eZ.UserMenuView} the view itself
+         */
+        render: function () {
+            this.get('container').addClass(CLASS_HIDDEN).addClass(CLASS_USER_MENU);
+
+            this._renderMenuItems();
+
+            return this;
+        },
+
+        /**
+         * Toggles the displayed value.
+         *
+         * @method toggleDisplayed
+         * @public
+         */
+        toggleDisplayed: function () {
+            this.set('displayed', !this.get('displayed'));
+        },
+
+        /**
+         * Toggles the visibility of a user menu.
+         *
+         * @method _toggleUserMenu
+         * @protected
+         * @param event {Object} event facade
+         */
+        _toggleUserMenu: function (event) {
+            var userMenu = this.get('container');
+
+            if (event.newVal) {
+                userMenu.removeClass(CLASS_HIDDEN);
+            } else {
+                userMenu.addClass(CLASS_HIDDEN);
+            }
+        },
+
+        /**
+         * Set the click outside event listener
+         *
+         * @method _setClickOutsideUserMenuHandler
+         * @protected
+         * @param event {Object} event facade
+         */
+        _setClickOutsideUserMenuHandler: function (event) {
+            if (event.newVal) {
+                this._clickOutsideUserMenuHandler = this.get('container').on('clickoutside', this._hideMenuOnClickOutside, this);
+            } else {
+                this._clickOutsideUserMenuHandler.detach();
+            }
+
+            return this;
+        },
+
+        /**
+         * Checks if clicked outside the user menu container and hides a user menu if it is visible.
+         *
+         * @method _hideMenuOnClickOutside
+         * @protected
+         */
+        _hideMenuOnClickOutside: function (event) {
+            if (!event.target.hasClass(CLASS_USER_PROFILE) && event.target.ancestor(SELECTOR_USER_PROFILE) === null && this.get('displayed')) {
+                this.set('displayed', false);
+            }
+        },
+
+        /**
+         * Renders menu items views
+         *
+         * @method _renderMenuItems
+         * @protected
+         */
+        _renderMenuItems: function () {
+            var menuItemsContainer = this.get('container'),
+                items = this.get('items');
+
+            items.sort(function (a, b) {
+                return b.get('priority') - a.get('priority');
+            });
+
+            items.forEach(Y.bind(function (view) {
+                view.addTarget(this);
+                menuItemsContainer.append(view.render().get('container'));
+                view.set('active', true);
```

Comment:
as I wrote in a previous comment, you should not set the `active` flag while rendering. Instead, the UserMenuView should have a `activeChange` handler that forwards the `active` value to the items.